Makefile

```diff
@@ -3,8 +3,8 @@ GOARCH ?= $(shell go env GOARCH)
 ARCH ?= $(shell uname -m)
 TARGET_IP ?= 127.0.0.1
 API_LEVEL ?= 29
+MOD_VERSION ?= 0.0.1
 
-UNAME=$(shell uname)
```

Comment:
I think that without this, the makefile will always fall through to the `else` clause below

(and `GOOS=android make build` fails on my linux laptop with 'not found')